jmh-core/src/main/java/org/openjdk/jmh/profile/MemPoolProfiler.java

```diff
@@ -65,8 +66,15 @@ public Collection<? extends Result> afterIteration(BenchmarkParams benchmarkPara
             results.add(new ScalarResult("mempool." + bean.getName() + ".used", used / BYTES_PER_KIB, "KiB", AggregationPolicy.MAX));
         }
 
+        for (BufferPoolMXBean bean : ManagementFactory.getPlatformMXBeans(BufferPoolMXBean.class)) {
+            final long used = bean.getMemoryUsed();
+            sum += used;
+            results.add(new ScalarResult(String.format("mempool.%s.used", bean.getName()), used / BYTES_PER_KIB, "KiB", AggregationPolicy.MAX));
```

Comment:
Please match the style of the for-loop above: 
 - drop `final` from `used`;
 - use regular concat instead of String.format;

Reply:
Done